src/main/scala/org/broadinstitute/hail/driver/IBDCommand.scala

```diff
@@ -22,6 +22,14 @@ object IBDCommand extends Command {
       usage = "Output TSV file for the IBD matrix")
     var outputFile: String = _
 
+    @Args4jOption(required = false, name = "--min",
+      usage = "Sample pairs with a PI_HAT below this value will not be included in the output. Must be in [0,1]")
+    var min: Double = -1
```

Comment:
I think you can use java.lang.Double here, which lets you use `Option(options.min)` in run.